src/go/rpk/pkg/tuners/disk/block_devices.go

```diff
@@ -216,11 +216,10 @@ func (b *blockDevices) getDeviceControllerPath(
 	splitSystemPath := strings.Split(devSystemPath, "/")
 	controllerPathParts := append([]string{"/"}, splitSystemPath[0:4]...)
 	pattern, _ := regexp.Compile(
-		"^[0-9ABCDEFabcdef]{4}:[0-9ABCDEFabcdef]{2}:[0-9ABCDEFabcdef]{2}\\.[0-9ABCDEFabcdef]$")
+		`^[0-9ABCDEFabcdef]{4}:[0-9ABCDEFabcdef]{2}:[0-9ABCDEFabcdef]{2}\.[0-9ABCDEFabcdef]$`)
 	for _, systemPathPart := range splitSystemPath[4:] {
+		controllerPathParts = append(controllerPathParts, systemPathPart)
```

Comment:
Would be good to add aa test in `block_devices_test.go` for the new case

Reply:
I don't really understand this change, nor what system paths look like up front. The original code here potentially could be clearer, and I wonder if it could be simpler.

I don't want to block this PR since it fixes a bug. I'm curious about the underlying behavior but can't look into it at the moment.

Reply:
@twmb for sure. These are probably some of the first lines of code that were merged into the repo, before I started at Vectorized.